### DISCUSSION TITLE
Motor synchronisation

### DISCUSSION THREAD

**Discussion by jacobcxdev** [ev3dev-lang-java/ev3dev-lang-java#799] [General]
I've been migrating my LeJOS project to ev3dev-lang-java because it requires the use of language features that were introduced post-1.7. One feature of LeJOS' motor regulation which my project pretty much depends on is motor synchronisation: the ability to start/stop motors at exactly the same time. Is there any reason that this isn't implemented in ev3dev-lang-java?

**Comment by jabrena** [comment]
Hi @jacobcxdev, 

Sorry the delay.

The main reason is because in LeJOS for EV3, that feature was implemented at runtime level. I had few discussion at EV3Dev, but that feature was paused and... in some moment appear in PyBricks but not at EV3Dev for general usage. I believe that the motor synchronisation could be implemented at Java level but it could be need some effort but It could be possible.

Juan Antonio

**Comment by flemmii** [comment]
Hi,
the synchronisation is implemented now, right? Or is it just the methods, but they don't work?

**Reply by jabrena** [reply]
No, it was not implemented
